packages/fuels-types/src/lib.rs

```diff
@@ -3,12 +3,17 @@
 //! We declare these in a dedicated, minimal crate in order to allow for downstream projects to
 //! consume or generate these ABI-compatible types without needing to pull in the rest of the SDK.
 
+extern crate core;
```

Comment:
are you sure you need to take the core crate (iiuc it's the `fuels-core` crate right?). 
Isn't that a circular deps? We're importing `fuels-types` everywhere including `fuels-core`.

Reply:
that somehow landed there while merging, good catch